libats-messaging/src/main/scala/com/advancedtelematic/libats/messaging/daemon/MessageBusListenerActor.scala

```diff
@@ -83,7 +83,7 @@ object MessageBusListenerActor {
   private lazy val log = LoggerFactory.getLogger(this.getClass)
 
   def loggingSink[M](implicit ml: MessageLike[M]): Sink[M, Future[Done]] = Sink.foreach[M] { msg =>
-    log.info(s"Processed ${ml.streamName} - ${ml.id(msg)}")
+    log.debug(s"Processed ${ml.streamName} - ${ml.id(msg)}")
```

Comment:
I think this might be a better default? Else we're gonna fill Splunk with a log of each event...